docs/modules/community/pages/mailing-lists.adoc

```diff
@@ -0,0 +1,165 @@
+
+*Watch where you are sending email*
+
+The majority of our mailing lists have set the Reply-To to go back to the list. That means that when you Reply to a
+message, it will go to the list and not to the original author directly. The reason is because it helps facilitate
+discussion on the list for everyone to benefit from. Be careful of this as sometimes you may intend to reply to a
+message directly to someone instead of the entire list. The appropriate contents of the Reply-To header is an age-old
+debate that should not be brought up on the mailing lists. You can examine opposing points of view condemning our
+convention and condoning it. Bringing this up for debate on a mailing list will add nothing new and is considered off-topic.
+
+*Do not cross post messages*
+
+In other words, pick one mailing list and send your messages to that mailing list only. Do not send your messages to
+multiple mailing lists. The reason is that people may be subscribed to one list and not to the other. Therefore, some
+people will only see part of the conversation.
+
+== James mailing lists
+
+=== Server user list
+
+*Low traffic*
+mailto:server-user-subscribe@james.apache.org[Subscribe]
+mailto:server-user-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/server-user@james.apache.org[Archive]
+
+This is the list where users of the Apache James (Server) meet and discuss issues. Developers are also expected to be
+subscribed to this list to offer support to users of Apache James (Server).
+
+=== Server developer list
+
+*High traffic*
+mailto:server-dev-subscribe@james.apache.org[Subscribe]
+mailto:server-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/server-dev@james.apache.org[Archive]
+
+This is the list where participating developers of the Apache James Project meet and discuss issues, code changes/additions,
+etc. Subscribers to this list get notices of each and every code change, build results, testing notices, etc. Do not
+send mail to this list with usage questions or configuration problems -- that's what server-user@james is for.
+
+=== Mime4j
+
+*Medium traffic*
+mailto:mime4j-dev-subscribe@james.apache.org[Subscribe]
+mailto:mime4j-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://mail-archives.apache.org/mod_mbox/james-mime4j-dev[Archive]
+
+Discussions on the http://james.apache.org/mime4j[Mime4j] parser library.
+
+=== Site developer list
+
+*Low traffic*
+mailto:site-dev-subscribe@james.apache.org[Subscribe]
+mailto:site-dev-unsubscribe@james.apache.org[Unsubscribe]
+http://www.mail-archive.com/site-dev@james.apache.org[Archive]
+
+This is the list where participating developers of the Apache James project discuss changes/additions to the James
+websites etc. Subscribers to this list get notifications of every commit of the website resources. Do not send mail to
+this list with James software problems -- that's what server-user@james is for.
```

Comment:
Should we keep that mailing list?

Reply:
If in doubt... my suggestion would be to remove it. IMO there are too many mailing lists. 😀

Also, since we're on the topic... I wonder if it would be possible to separate the notifications from posts by members. Like maybe: notifications@james.apache.org or whatever. wdyt?

Reply:
That is what general is for, apparently.  You are very right.

Reply:
I would also go for notifications on separate ML . 
We should have a separate channel for user vs developer discussions and make it clear in the docs.
Right now gitter for example has all discussions into once place.

Reply:
I would agree on separating notifications to an other ML. It's hard sometimes to distinguish user mails from notifications in server-dev mailing list for example

@ieugen : not sure to get your point with Gitter though, to me it seems gitter discussions are all user oriented?

Reply:
I'll add a JIRA issue and make sure we get notifications@james.apache.org and then make the changes for the projects so we have both commits and JIRA comments ?! sent to notifications.

The main benefit is that commits tend to polute the ML and will create a lot of traffic for archive as well.

Regarding Gitter, I feel the need to talk to fellow developers on something else than ML - so we can have a quick discussion and then send the conclusions to ML.

Reply:
See https://issues.apache.org/jira/browse/INFRA-20735 , please comment there.